### PR TITLE
[WIP]Update wiki with new dts and filters

### PR DESCRIPTION
**Pull request by MARTINI1**
Add new DTS and filters from PRs:
https://github.com/PokeAlarm/PokeAlarm/pull/563
https://github.com/PokeAlarm/PokeAlarm/pull/565
https://github.com/PokeAlarm/PokeAlarm/pull/566
https://github.com/PokeAlarm/PokeAlarm/pull/568

also removed the not used sections of wiki: boxcar, pushover ~and docker~

add PRs:
https://github.com/PokeAlarm/PokeAlarm/pull/570
https://github.com/PokeAlarm/PokeAlarm/pull/571

recovered docker and added in wiki and sidebar

add changes of PR:
https://github.com/PokeAlarm/PokeAlarm/pull/564

add note for a issue:
https://github.com/PokeAlarm/PokeAlarm/issues/569

### CODE REVIEW

File: Raid-DTS.md [hunk 1]
```diff
@@ -34,6 +41,10 @@ This page assumes:
 | weather          | Weather name of the raid.                               |
 | weather_or_empty | Weather name of the raid, or empty string if unknown.   |
 | weather_emoji    | Weather emoji of the raid, or empty string if unknown.  |
+| boosted_weather_id | Return weather ID if monster is boosted.              |
+| boosted_weather  | Return weather name if monster is boosted.              |
+| boosted_weather_or_empty | Return weather name if monster is boosted, or empty string if unknown. |
```
Comment (kvangent) [inline Raid-DTS.md]: This is getting removed.

File: Telegram.md [hunk 2]
```diff
@@ -75,25 +89,33 @@ Parameters at the alarm level will be the default to alert-level parameters.
...
+Below is how an advanced alarm configuration would appear in `alarms.json`.
+Note that this is **not** the entire `alarms.json`, but only the section
+pertaining to the alarm portion of the JSON file.
 
 ```json
 {
-    "active":"True",
-    "type":"telegram",
+  "telegram_alarm":{
+    "active":true,
```
Comment (kvangent) [inline Telegram.md]: You still need type

File: Telegram.md [hunk 3]
```diff
@@ -75,25 +89,33 @@ Parameters at the alarm level will be the default to alert-level parameters.
...
     "bot_token":"YOUR_BOT_TOKEN",
     "chat_id":"YOUR_CHAT_ID",
     "map_notify":"False",
```
Comment (kvangent) [inline Telegram.md]: We should try to encourage proper json types (eg. `false` instead of `"False`)

Comment (MARTINI1) [reply Telegram.md]: I have changed all "False"/"True" to false/true

**Comment by MARTINI1** [comment]
@kvangent I think everything is already to merge

**Pull request merged**